server/views/v2Manage/outOfServiceBeds/show.njk

```diff
@@ -6,6 +6,8 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
```

Comment:
I think a lot of these imports will be redundant now, could we remove the unused ones?

Reply:
good point!